src/components/PeopleTable.tsx

```diff
@@ -1,4 +1,28 @@
-export const PeopleTable = () => {
+import React, { useState } from 'react';
+import { Person } from '../types';
+import { SortType, PropName } from './enum';
```

Comment:
It's better to place the 'enum' folder inside the 'types' folder, as it represents type information.